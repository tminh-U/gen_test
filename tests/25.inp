12
1 10 7 4 15 4 6 9 5 18 20 14 
15
9 11
4 11
4 6
5 12
2 3
10 11
7 10
1 5
4 6
10 10
12 12
12 12
8 8
2 8
2 8
