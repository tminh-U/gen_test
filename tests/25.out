8
88
6
54
3
4
10
16
6
2
2
2
2
23
23
